examples/mobile/UIComponents/components/appbar/more-option-list-max-width.html

```diff
@@ -37,8 +37,11 @@
 		</div>
 
 		<script>
+			var dropDownMenu = document.getElementById("dropdownmenu");
+			dropdownmenu.selectedIndex = -1;
```

Comment:
is this line necessary? items in example don't have selected attribute set

Reply:
Yes, it's needed. By default, select element shows selection for first element (index 0). This is a way not to do this.